x-pack/plugins/observability_solution/observability/public/components/rule_condition_chart/painless_tinymath_parser.test.ts

```diff
@@ -193,12 +289,36 @@ describe('PainlessTinyMathParser', () => {
     const parser = new PainlessTinyMathParser({
       equation,
       aggMap: {
-        A: 'average(system.cpu.system.pct)',
-        B: 'average(system.cpu.user.pct)',
-        C: 'average(system.cpu.total.pct)',
-        D: 'average(system.cpu.cores)',
-        E: 'count()',
-        F: 'sum(system.cpu.total.pct)',
+        A: {
+          operationWithField: 'average(system.cpu.system.pct)',
+          operation: 'average',
+          sourceField: 'system.cpu.system.pct',
+        },
+        B: {
+          operationWithField: 'average(system.cpu.user.pct)',
+          operation: 'average',
+          sourceField: 'system.cpu.user.pct',
+        },
+        C: {
+          operationWithField: 'average(system.cpu.total.pct)',
+          operation: 'average',
+          sourceField: 'system.cpu.total.pct',
+        },
+        D: {
+          operationWithField: 'average(system.cpu.cores)',
+          operation: 'average',
+          sourceField: 'system.cpu.cores',
+        },
+        E: {
+          operationWithField: 'count(___records___)',
+          operation: 'count',
+          sourceField: '___records___',
+        },
```

Comment:
before we where passing just `count()` instead of `count(__records__)`. I see where this comes from, but I'm wondering if we need to keep it as it was in the previous test.

Reply:
@markov00 Good catch! If I am not mistaken, before we needed to pass `___records___` to count aggregation in Lens visualization, but I see it is not needed anymore. Has there been any recent change regarding this in Lens?

This test was using `count()` but I think it should have been `count(___records___)`. I have now removed usage of `___records___` from rule condition chart and tests in this [commit](https://github.com/elastic/kibana/pull/199643/commits/2dffb98fc829598cfd399847d91951d4fcfe6191).